src/playground-helper.ts

```diff
@@ -0,0 +1,26 @@
+import {compressToEncodedURIComponent} from 'lz-string'
+
+function unindent(value: string) {
+  // remove white spaces first, to save a few bytes.
+  // testing-playground will reformat on load any ways.
+  return value.replace(/[ \t]*[\n][ \t]*/g, '\n')
+}
+
+function encode(value: string) {
+  return compressToEncodedURIComponent(unindent(value))
+}
+
+function getPlaygroundUrl(element: Element) {
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+  if (!element) {
```

Comment:
```suggestion
function getPlaygroundUrl(element: Element | null) {
  if (element === null) {
```

Let's match runtime and static types

Reply:
I touched upon this a little bit: https://github.com/testing-library/dom-testing-library/pull/852#discussion_r557631676
Do you still think we should match the types, given that `null` is an erroneous type/value to pass in?

Reply:
See https://github.com/testing-library/dom-testing-library/pull/852#discussion_r646339965

internal methods: static type checking
public methods: static type checking + extraneous runtime type checking for users not using static type checking

Reply:
Since the runtime checks has been moved to `logTestingPlaygroundURL` I was able to remove the `if (element === null)` from here :-)